importers/exchanges/kraken.js

```diff
@@ -43,9 +43,8 @@ var handleFetch = (err, trades) => {
         return fetcher.emit('trades', []);
     }
 
-    var last = moment.unix(_.last(trades).date);
+    var last = moment.unix(_.last(trades).date).utc();
```

Comment:
you mentioned some bugs related to UTC, do you have more information regarding what this exactly fixes?

Reply:
This helped me to get --imports completely, otherwise 2h were missing. UTC vs. GMT+2 prob.